workers/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "workers"
+version = "0.1.0"
+edition = "2021"
+
+[[bin]]
+name = "dvn"
+path = "src/bin/dvn.rs"
+
+[dependencies]
```

Comment:
some of these dependencies are workspace level dependencies. lets get them in the root Cargo.toml